proto/bitcoin/drivechaind/v1alpha/drivechain.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package bitcoin.drivechaind.v1alpha;
```

Comment:
Let's just go with `v1`, `v1alpha` is such a hassle to type all the time

Reply:
okidoki, also change the bitcoind package?

Reply:
no, that's a breaking change i don't want to deal with